common/lib/capa/capa/capa_problem.py

```diff
@@ -561,22 +561,22 @@ def find_question_label(self, answer_id):
             a string with the question text
         """
 
-        def log_error():
+        def generate_default_question_label():
             """
-            Temporary method for logging
+            To create question string like "Question 2" by adding "Question" and its position number.
```

Comment:
For reference 👍 https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings